client/blocks/comments/form.scss

```diff
@@ -59,22 +59,18 @@
 	button {
 		opacity: 0;
 		position: absolute;
-		top: 4px;
-		right: 16px;
-		padding: 4px;
-		font-size: $font-body-extra-small;
-		font-weight: 600;
-		text-transform: uppercase;
-	color: var( --color-text-subtle );
+		float: right;
```

Comment:
Absolute positioning and floats don't go together. I'm not familiar with the code but I assume you'd want the float to be part of the `&.is-visible` class where you're changing the positioning to static.

Reply:
Absolute is just to remove it from effecting the presentation of other elements while it's hidden. `position: static;` is applied while `is-visible` to remove the absolute positioning. So, in effect, while it's presented, it won't have both absolute positioning and the float.

Reply:
I get what you mean though. `float: right;` has no effect while `position: absolute;` is applied. I'm not sure which one I prefer, moving it over to `is-visible` to make that connection to `position: static;` more obvious, or leaving it on the base `button` here in order to suggest what the default positioning should be as you're reading through.

Reply:
After talking about this more, I'm on @frontdevde's side on this. Let's move `float: right;` over to `&.is-visible` since `float: right;` has no effect until `position: absolute;` is replaced with `position: static;`.